launch_testing/launch_testing/__init__.py

```diff
@@ -22,8 +22,13 @@
 from launch.event_handlers import OnProcessIO
 from launch.events import Shutdown
 
+from . import actions
 from .output import create_output_check
 
+__all__ = [
+    'actions',
+]
+
```

Comment:
@ivanpauno I'm not sure we want __all__ to behave like this (see [here](http://xion.io/post/code/python-all-wild-imports.html)). Maybe we should add the class defined below and the `output` module as well?